src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -239,7 +239,8 @@ def _rbd_image(cls, ioctx, pool_name, namespace, image_name):
 
             stat = img.stat()
             stat['name'] = image_name
-            stat['id'] = img.id()
+            stat['id'], stat['image_format'] = (img.id(), 2) if \
+                not img.old_format() else (stat['block_name_prefix'], 1)
```

Comment:
Nit: Technically, v1 images don't have an id. So long as this isn't being used as a key for something and is mearly for display purposes, I'd say just drop the `rb.` prefix from the v1 block name prefix.

Reply:
So, cannot that be expected to be a unique index? The Dashboard front-end table needs a unique id to work. If not the `block_name_prefix`, what else could do the trick?

Reply:
It's guaranteed to be unique per pool -- it just is meaningless to the user. If you just need something for a unique index for internal structures, definitely can use it.

Reply:
Great, thanks! For the ID v1 I'll concat `pool_id` and `block_name_prefix` then (Dashboard RBD images provides an aggregate view across pools, so IDs need to be globally unique).

Reply:
That applies for v2 images as well -- ids are only guaranteed to be unique within the same pool + namespace.

Reply:
Done, concatenating pool ID and image ID:
- v1: `id: "2.rb.0.104e.6b8b4567"`
- v2: `id: "2.104c2fe7ebe3"`

Reply:
> That applies for v2 images as well -- ids are only guaranteed to be unique within the same pool + namespace.

Do we want to insert the namespace string into the id string for v2 images as well?

Reply:
> Do we want to insert the namespace string into the id string for v2 images as well?

Indeed -- good catch.

Reply:
Thanks @bk201 ! Done.